test/Mono.Linker.Tests.Cases/DataFlow/ArrayDataFlow.cs

```diff
@@ -226,8 +226,8 @@ static void TestArrayResetStoreUnknownIndex (int i = 0)
 
 		// https://github.com/dotnet/linker/issues/2680 - analyzer doesn't reset array in this case
 		[ExpectedWarning ("IL2062", nameof (DataFlowTypeExtensions.RequiresPublicFields), ProducedBy = ProducedBy.Trimmer)]
-		// https://github.com/dotnet/linker/issues/2632 - Ref params don't reset or track.
-		// [ExpectedWarning ("IL2062", nameof (DataFlowTypeExtensions.RequiresPublicMethods))]
+		// https://github.com/dotnet/linker/issues/2680 - analyzer doesn't reset array in this case
+		[ExpectedWarning ("IL2062", nameof (DataFlowTypeExtensions.RequiresPublicMethods), ProducedBy = ProducedBy.Trimmer)]
 		static void TestArrayResetGetElementOnByRefArray (int i = 0)
 		{
 			Type[] arr = new Type[] { typeof (TestType), typeof (TestType) };
```

Comment:
Nit: below there's comment that "linker doesn't" warn - I think with this change it now does.